### DISCUSSION TITLE
Hire multiple carriers

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#309] [General]
Hi!

I'm sure you've already thought about it, but as I can't find any related discussion, I take the liberty of asking the question:

It would be really great to be able to hire multiple carriers per granary and per warehouse to transport more resources.

And maybe hiring additional employees in the markets? for a better supply, and on farms, to solve the problem of distances?

**Comment by crudelios** [comment]
Check #295 for the solution we're working on 😉 

Edit: I see the link isn't too descriptive. It's basically a cart depot where you can select a source, a destination and a condition and a cartpusher will spawn and move goods from source to desination when the condition is met. The idea is for it to be small so that many depots can be added to your city, therefore helping with the flow of goods between different city sections.

**Reply by mathmaj** [reply]
Thanks you very much for your answer! :) This will be great!
